src/Components/Footer/Footer.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+import { FilterParams } from '../../types/FilteredParams';
+
+type Props = {
+  todos: Todo[];
+  filterValue: FilterParams,
+  setFilterValue: (filterValue: FilterParams) => void;
+  removeCompletedTodos: () => void;
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  filterValue,
+  setFilterValue,
+  removeCompletedTodos,
+}) => {
+  const isSomeCompletedTodo = todos.some(todo => todo.completed);
+  const activeTodos = todos.filter(todo => !todo.completed);
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${activeTodos.length} items left`}
+      </span>
+
+      <nav className="filter" data-cy="Filter">
```

Comment:
You can use this logic in another component where you can write only code with select. And then use this component in component ```Footer```